scripts/get-usage/get-all-components.ts

```diff
@@ -72,27 +73,38 @@ export const getAllComponentsAndPackages = async () => {
 	const packages: Record<string, string | string[]> = {
 		[packageNames.brand]: paths.brand,
 		[packageNames.icons]: paths.icons,
-		[packageNames.kitchen]: await getKitchenComponentPaths(),
 		...(await getComponentPackageNamesWithPaths()),
+		'@guardian/source-react-components': join(
```

Comment:
Should we add `@guardian/source-react-components` to `packageNames`?

Reply:
Yes, we totally should. This was me being lazy!